modules/os2forms_nemid/src/Plugin/WebformElement/NemidChildrenSelect.php

```diff
@@ -136,23 +134,30 @@ public function handleElementPrepopulate(array &$element, FormStateInterface &$f
 
     $options = [];
 
+    $showAddressNameProtectionMessage = FALSE;
+
     if ($cprLookupResult) {
       $prepopulateKey = $this->getPrepopulateFieldFieldKey($element);
       if ($children = $cprLookupResult->getFieldValue($prepopulateKey)) {
         if (is_array($children) && !empty($children)) {
           foreach ($children as $child) {
-            if ($element['#cpr_output_type'] == 'cpr') {
-              $options[$child['cpr']] = $child['name'];
+            if ($child['nameAddressProtected']) {
+              $options[$child['cpr']] = $child['cpr'] . ' (Navne- og adressebeskyttet)';
```

Comment:
“(Navne- og adressebeskyttet)” should be translatable (OS2Forms support multi-lingual forms).